setup.cfg

```diff
@@ -4,3 +4,9 @@ ignore = C901
 
 [pylama:pep8]
 max_line_length = 100
+
+[pylama:pycodestyle]
+max_line_length = 100
+
+[pylama:pylint]
+max_line_length = 100
```

Comment:
Cannot we define a single pylama section with global settings? I feel it weird that we have to do it for each individual checker.